stable/neo4j/values.yaml

```diff
@@ -7,7 +7,7 @@ name: "neo4j"
 
 # Specs for the Neo4j docker image
 image: "neo4j"
-imageTag: "3.2.3-enterprise"
+imageTag: "3.3.4-enterprise"
```

Comment:
Could it be the CE by default?

Reply:
@ant31 clustering isn't in the CE version so it only makes sense to use Helm with the EE version.